common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -3915,6 +3917,17 @@ object CelebornConf extends Logging {
       .booleanConf
       .createWithDefault(false)
 
+  val WORKER_GRACEFUL_SHUTDOWN_DB_DELETE_FAILURE_POLICY: ConfigEntry[String] =
+    buildConf("celeborn.worker.graceful.shutdown.dbDeleteFailurePolicy")
```

Comment:
```suggestion
    buildConf("celeborn.worker.graceful.shutdown.deleteDbFailurePolicy")
```

Reply:
Here it should be that db deletes a certain key, not delete db.